src/Poly.jl

```diff
@@ -1975,6 +1991,174 @@ function primpart(a::PolyElem)
    end
 end
 
+###############################################################################
+#
+#   Half GCD
+#
+###############################################################################
+
+# these two functions determine cutoffs
+function _hgcd_prefers_basecase(a, b)
+   return true
+end
+
+function _mat22_mul_prefers_classical(a11, a12, a21, a22, b11, b12, b21, b22)
+   return true
```

Comment:
Is this the function that you referred to (the one that cannot be tested)?

Reply:
The test code now redefines it